.eslintrc

```diff
@@ -1,5 +1,13 @@
 {
   "parser": "babel-eslint",
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "parserOptions": {
+    "ecmaVersion": 2020
```

Comment:
Wouldn't this also require `babel-preset-es2020` in order to be useful ?

Reply:
Removed it entirely